docs/source/en/_toctree.yml

```diff
@@ -157,6 +157,8 @@
         title: Pix2Pix Zero
       - local: api/pipelines/stable_diffusion/self_attention_guidance
         title: Self-Attention Guidance
+      - local: api/pipelines/stable_diffusion/multiDiffusion_panorama
+        title: MultiDiffusion Panorama
```

Comment:
I think we are missing changes to `overview.mdx` and a standalone doc for the pipeline (which should be named `multidiffusion_panorama.mdx`).

Reply:
Seems like the doc page is missing.